src/platform/backends/shared/base_virtual_machine.cpp

```diff
@@ -541,6 +541,7 @@ void BaseVirtualMachine::restore_snapshot(const std::string& name, VMSpecs& spec
     specs.num_cores = snapshot->get_num_cores();
     specs.mem_size = snapshot->get_mem_size();
     specs.disk_space = snapshot->get_disk_space();
+    const bool is_extra_interfaces_different = specs.extra_interfaces != snapshot->get_extra_interfaces();
```

Comment:
I'd change this to `are_extra_interfaces_different`. I know it's a data structure, but it sounds better to me to have the plural.

Reply:
Yep, that is a better name. Will do.